PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -392,25 +400,28 @@ struct FlowGFWPbPb {
     }
 
     auto multNTracksPV = collision.multNTracksPV();
+    auto occupancy = collision.trackOccupancyInTimeRange();
 
     if (centrality >= 70. || centrality < 0)
-      return false;
+      return 0;
     if (abs(vtxz) > cfgCutVertex)
-      return false;
+      return 0;
     if (multNTracksPV < fMultPVCutLow->Eval(centrality))
-      return false;
+      return 0;
     if (multNTracksPV > fMultPVCutHigh->Eval(centrality))
-      return false;
+      return 0;
     if (multTrk < fMultCutLow->Eval(centrality))
-      return false;
+      return 0;
     if (multTrk > fMultCutHigh->Eval(centrality))
-      return false;
+      return 0;
+    if (occupancy < cfgCutOccupancyLow || occupancy > cfgCutOccupancyHigh)
+      return 0;
 
     // V0A T0A 5 sigma cut
     if (abs(collision.multFV0A() - fT0AV0AMean->Eval(collision.multFT0A())) > 5 * fT0AV0ASigma->Eval(collision.multFT0A()))
-      return false;
+      return 0;
 
-    return true;
+    return 1;
```

Comment:
Please, for the next iteration, keep the `bool` returns
Your function returns a boolean and it is more readable if `true` and `false` are in place and not `0` or `1`

Reply:
Ahh my bad. I remember that you had mentioned this in the past. I will fix that in the next iteration, thanks!